package.json

```diff
@@ -10,15 +10,25 @@
     "type": "git",
     "url": "https://github.com/webcomponents/template.git"
   },
-  "author": "The Polymer Authors",
+  "author": "The Polymer Project Authors (https://polymer.github.io/AUTHORS.txt)",
```

Comment:
I agree with changing "Polymer Project Authors" to "Polymer Authors", but what's wrong with using URL to authors list in `bower.json`/`package.json`?

@azakus, WDYT?